stable/artifactory-ha/templates/artifactory-installer-info.yaml

```diff
@@ -10,16 +10,13 @@ metadata:
 data:
   installer-info.json: |
     {
-      "productId": "JFrogInstaller_Helm_{{ template "artifactory-ha.chart" . }}",
+      "productId": "Helm_artifactory-ha/{{ .Chart.Version }}",
       "features": [
       {
-        "featureId": "AppVersion/{{ default .Chart.AppVersion .Values.artifactory.image.version }}"
+        "featureId": "ArtifactoryVersion/{{ default .Chart.AppVersion .Values.artifactory.image.version }}"
       },
       {
-        "featureId": "Installer/{{ default "helm" .Values.installer.type }}"
-      },
-      {
-        "featureId": "Platform/{{ default "kubernetes" .Values.installer.platform }}"
```

Comment:
Keep this as its used in different fork of chart repo for other platforms like GKE-MP, Rancher, IBM

Reply:
@ankushchadha Can give you more context.

Reply:
These changes were requested by @freddy33 .
@freddy33 - do you want to comment here?

Reply:
one of the example -> https://github.com/IBM/charts/blob/master/community/artifactory-ha/values.yaml#L11

Reply:
Ok. `Platform` I'll add again. Do we really have use for the `Installer/`?

Reply:
That you can remove.